packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -257,7 +277,7 @@ const RemoteVideoTile = React.memo(
       }
 
       return <StreamMedia videoStreamElement={renderElement} />;
-    }, [renderElement, renderElement?.childElementCount]);
+    }, [renderElement]);
```

Comment:
Why does the useMemo no longer need to update on childElementCount?

Reply:
We are watching `renderElement` and `renderElement?.childElementCount` becomes redundant

Reply:
The useMemo (I'm assuming) does a shallow check so will only check the renderElement reference not the underlying properties (so childElementCount could change but this won't rerender).

Reply:
Although, I tested it and it works. Still reverting it just to be safe.